### PR TITLE
save lang preference

### PR DESCRIPTION
**Pull request by makoteq**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [ ] Added Screenshot / Path to tile to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at : https://makoteq.github.io/labyrinth.com/ :smile:

- Fixes <Add issue number here> https://github.com/fossasia/labyrinth/issues/1071

### The problem I want to solve or the facility I want to improve is/are
**Unnecessary user interaction** - User in every enter to the site have to choose language preferences

### My steps to solve it
I created File 'selectLang.js' in which I checked the 'lang' variable in localStorage. 
If  not existed -> language selection menu is shown.

In the  translate.js file, in the function callable after click on 'select_language' class item, I added code  to store selected language value in localStorage.

### Screenshots,Path to Tile

no design changes :thinking:

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by makoteq** [comment]
Link to preview: https://makoteq.github.io/labyrinth.com/

File: js/selectLang.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+
+window.onload=()=>{
+  // localStorage.clear();
+   // localStorage.setItem('lang', lang);
+  /*  localStorage.setItem('lang', selectWindow);
```
Comment (YashKumarVerma) [inline js/selectLang.js]: I think we need only one comment here

File: js/selectLang.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    
+            localStorage.setItem('lang', lang);
+            alert(localStorage.getItem('lang'));
+    
+        });
+        $('#select-language').hide();
+      }else   $('#select-language').show();
+      $(function () {
+
+});*/
```
Comment (YashKumarVerma) [inline js/selectLang.js]: Unwanted code should not be committed

**Comment by makoteq** [comment]
@YashKumarVerma Check now! :smile:

File: js/selectLang.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
+window.onload=()=>{
```
Comment (YashKumarVerma) [inline js/selectLang.js]: [Build Errors](https://travis-ci.org/fossasia/labyrinth/builds/625009039#L535)

**Comment by makoteq** [comment]
@YashKumarVerma I dont know what's wrong. Can you help? please

File: js/selectLang.js (hunk 3 above)
Comment (YashKumarVerma) [reply js/selectLang.js]: https://travis-ci.org/fossasia/labyrinth/builds/625054805#L506 
Fixed.

**Comment by YashKumarVerma** [comment]
There are no errors in the javascript code.

https://travis-ci.org/fossasia/labyrinth/builds/625054805#L506

**Comment by YashKumarVerma** [comment]
However, we have this error:

image

**Comment by makoteq** [comment]
@YashKumarVerma  I dont see these errors in my console

**Comment by makoteq** [comment]
@YashKumarVerma Can you help?

**Comment by YashKumarVerma** [comment]
> @YashKumarVerma Can you help?

there seems to be an error in the code. can you try fixing it?

**Comment by makoteq** [comment]
@YashKumarVerma Can you check now?

**Comment by makoteq** [comment]
@YashKumarVerma  can you specify in which part of application this error show up?

**Comment by YashKumarVerma** [comment]
> @YashKumarVerma can you specify in which part of application this error show up?

as soon as page loads

**Comment by makoteq** [comment]
@YashKumarVerma Fix. 😅

**Comment by YashKumarVerma** [comment]
> @YashKumarVerma Fixed.

are you sure? I think i can still see the error.

**Comment by makoteq** [comment]
https://makoteq.github.io/labyrinth.com/ I dont see these errors I think its fixed

**Comment by YashKumarVerma** [comment]
kindly merge the commits into one.

**Comment by makoteq** [comment]
Can you explain how i can do this, please @YashKumarVerma

**Comment by makoteq** [comment]
Done @YashKumarVerma

**Comment by makoteq** [comment]
@YashKumarVerma what about now?

**Comment by makoteq** [comment]
@YashKumarVerma

**Comment by YashKumarVerma** [comment]
I'm not sure with the error, let's wait until someone else reviews it.

**Comment by makoteq** [comment]
> I'm not sure with the error, let's wait until someone else reviews it.
@YashKumarVerma 
Can you tag other mentors?

**Comment by YashKumarVerma** [comment]
@niccokunzmann @abishekvashok

**Comment by makoteq** [comment]
@YashKumarVerma 
I think nobody will answer

**Comment by niccokunzmann** [comment]
Ooops. Did I just merge?

**Comment by niccokunzmann** [comment]
It seems to work...

**Comment by makoteq** [comment]
@makoteq whats wrong?

**Pull request merged**